test/suite_test.go

```diff
@@ -47,10 +46,6 @@ var testEnv *envtest.Environment
 
 func TestAPIs(t *testing.T) {
 	RegisterFailHandler(Fail)
-
-	RunSpecsWithDefaultAndCustomReporters(t,
```

Comment:
0.13.0 -> 0.14.0 does not work because below package was dropped:
[sigs.k8s.io/controller-runtime/pkg/envtest/printer](http://sigs.k8s.io/controller-runtime/pkg/envtest/printer)

We are using this in Gingko tests. 
Reported issue: https://github.com/kubernetes-sigs/controller-runtime/issues/2113